EQL-Client-Project/src/pages/Category/Category.tsx

```diff
@@ -1,5 +1,17 @@
+import * as S from './Category.style';
+import CategoryNav from '../../components/Category/CategoryNav/CategoryNav';
+import CategoryMainContent from '../../components/Category/CategoryMainContent/CategoryMainContent';
+
 const Category = () => {
-  return <div>Category</div>;
+  return (<S.CategoryContainer>
+    <S.CategoryWrapper>
+      <CategoryNav></CategoryNav>
+      <CategoryMainContent></CategoryMainContent>
```

Comment:
요 두개는 가운데 들어가는게 없으면 셀프 클로징태그로 바꿔줘도 될 거 같아요~!
```suggestion
      <CategoryNav/>
      <CategoryMainContent/>
```